pynonymizer/strategy/parser.py

```diff
@@ -102,7 +103,7 @@ def __normalize_update_columns_list(columns_config):
             for column_name, column_config in columns_config.items():
                 normalized_column = StrategyParser.__normalize_column_config(column_config)
                 normalized_column["column_name"] = column_name
-                column_list.append( normalized_column )
+                column_list.append(normalized_column)
```

Comment:
Side note, sorry to clutter up the diff, but I noticed that there's mixed use of spaces after parens throughout the project. My editor complains about `PEP8: E201/E202` on these.

Reply:
😅the whole project needs a lint-up i think. Something I think we should spin out into another issue... but i welcome any targetted improvements  for PRs!